code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -46,4 +44,6 @@ const StageProgress = React.createClass({
     );
   }
 });
-module.exports = StageProgress;
+export default connect((state, ownProps) => ({
+  levels: ownProps.levels || state.stages[0].levels
```

Comment:
The issue I ran into is that I want to deeply-connect certain props (pull them from the store vs. pass them all the way down through each component).  Is there a way to do this without connecting all the intermediate components?